src/shared/ui/input/input.test.tsx

```diff
@@ -0,0 +1,36 @@
+import Input from './input'
+import { describe, it, expect, afterEach } from 'vitest'
+import { cleanup, render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { useForm } from 'react-hook-form'
+
+function InputTest() {
+  const { register, watch } = useForm()
+  const emailValue = watch('email')
+  return (
+    <>
+      <Input name="email" register={register} placeholder="이메일을 입력하세요" className="bg-blue-500" />
```

Comment:
```suggestion
      <Input {...register("email")} placeholder="이메일을 입력하세요" className="bg-blue-500" />
```

register에는 name값도 포함이여서요. 위처럼 하는게 더 나아보여요.